src/sensors/mod.rs

```diff
@@ -71,14 +99,32 @@ pub trait SensorTemplate: Send + Sync {
 /// Container for instances of all sensor types.
 #[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
 pub struct Sensors {
-    #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub people_now_present: Vec<PeopleNowPresentSensor>,
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
     pub temperature: Vec<TemperatureSensor>,
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub door_locked: Vec<DoorLockedSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub barometer: Vec<BarometerSensor>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub radiation: Option<RadiationSensors>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
     pub humidity: Vec<HumiditySensor>,
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub beverage_supply: Vec<BeverageSupplySensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
     pub power_consumption: Vec<PowerConsumptionSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub wind: Vec<WindSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub network_connections: Vec<NetworkConnectionsSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub account_balance: Vec<AccountBalanceSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub total_member_count: Vec<TotalMemberCountSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub people_now_present: Vec<PeopleNowPresentSensor>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub network_traffic: Vec<NetworkTrafficSensor>,
```

Comment:
Are the sensor now ordered in any specific order? Or why did you move `people_now_present`?

Reply:
The order is as per the [API docs](https://spaceapi.io/docs/).